### PR TITLE
feat(core-react): add typings to index.d.ts for Container, Grid, & Cell

### PR DESCRIPTION
**Pull request by Morgantheplant**
https://jira.weworkers.io/browse/WEB-815

adds typings for Container, Grid and Cell components

### CODE REVIEW

File: packages/core-react/index.d.ts [hunk 1]
```diff
@@ -8,4 +8,21 @@ export type TextFieldProps = {
...
+declare const Container: React.FC<ContainerProps>;
+
+export type GridProps = React.HTMLProps<HTMLElement>;
+
+declare const Grid: React.FC<GridProps>;
+
+export type CellProps = {
+  push?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+  span?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+} & React.HTMLProps<HTMLElement>;;
```
Comment (seimithwework) [inline packages/core-react/index.d.ts]: woops `;;` but I'll fix this for you!

File: packages/core-react/index.d.ts [hunk 2]
```diff
@@ -8,4 +8,21 @@ export type TextFieldProps = {
...
+
+declare const Cell: React.FC<CellProps>;
+
+export { Container, Grid, TextField };
```
Comment (seimithwework) [inline packages/core-react/index.d.ts]: And `Cell` haha

File: packages/core-react/index.d.ts (hunk 1 above)
Comment (Morgantheplant) [reply packages/core-react/index.d.ts]: good catch! Surprised this wasn't caught by a commit hook

File: packages/core-react/index.d.ts (hunk 2 above)
Comment (Morgantheplant) [reply packages/core-react/index.d.ts]: this was intentional: https://github.com/wework/ray/pull/213#discussion_r319401633 I am ok with exporting it though

Comment (seimithwework) [reply packages/core-react/index.d.ts]: Ahh ok cool!

**Pull request merged**